daemon/cmd/daemon.go

```diff
@@ -188,15 +188,11 @@ func (d *Daemon) init() error {
 		}
 
 		if option.Config.SockopsEnable {
-			disableSockops := func(err error) {
-				option.Config.SockopsEnable = false
-				log.WithError(err).Warn("Disabled '--sockops-enable' due to missing BPF kernel support")
-			}
 			eppolicymap.CreateEPPolicyMap()
 			if err := sockops.SockmapEnable(); err != nil {
-				disableSockops(err)
+				log.WithError(err).Error("Failed to enable Sockmap")
 			} else if err := sockops.SkmsgEnable(); err != nil {
-				disableSockops(err)
+				log.WithError(err).Error("Failed to enable Sockmap")
```

Comment:
This should be `Failed to enable Sockmsg`, right?

Reply:
Yes will fix it.

Reply:
@soumynathan line 198 should be `Failed to enable Sockmap`, line 200 should be `Failed to enable Skmsg`